### ISSUE TITLE
Support for FQCN-based calls to imported functions in `fully_qualified_strict_types`

### ISSUE THREAD

**Issue by Wirone** [PHP-CS-Fixer/PHP-CS-Fixer#7460]
## Feature request

Following [this comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5594#issuecomment-1824116639), we need to add support for imported functions to `fully_qualified_strict_types`.

```php
<?php

use function foo\bar;

\foo\bar();
```

should be changed to:

```php
<?php

use function foo\bar;

bar();
```

**Comment by mvorisek** [comment]
`use const x\y;` is working as expected?

**Comment by Wirone** [comment]
@mvorisek no, but probably it should be implemented separately to make code review easier.

**Comment by mvorisek** [comment]
after #7679 this is easy to fix

**Comment by mvorisek** [comment]
What is the state after https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7705?
